video/out/w32_common.c

```diff
@@ -1543,6 +1582,10 @@ static int gui_thread_control(struct vo_w32_state *w32, int request, void *arg)
         if (w32->opts->fullscreen != w32->current_fs)
             reinit_window_state(w32);
         return VO_TRUE;
+    case VOCTRL_GET_FULLSCREEN:
+        *(int *)arg = w32->toggle_fs ^ w32->current_fs;
+        reinit_window_state(w32);
```

Comment:
I think maybe it would be better if e.g. VOCTRL_FULLSCREEN copied the option value to toggle_fs, and having `reinit_window_state()` ignore the option value. I think that could possibly make the code a bit easier to follow.